### DISCUSSION TITLE
Sponsorship

### DISCUSSION THREAD

**Discussion by leesmith110** [jertel/elastalert2#894] [Ideas]
Hi there,

We use elastalert quite a lot and your new fork is something we are going to use going forwards.

With that in mind, do you have any GitHub sponsorship setup, would love to contribute in some fashion... [https://github.com/sponsors](https://github.com/sponsors)

**Comment by jertel** [comment]
Hello, there is not currently a sponsorship account in place. I've not seen sufficient community sponsorship contribution amounts to justify the overhead required to setup the sponsorship and annual tax filing/reporting and accounting requirements.

However, I appreciate your willingness to support opensource software! Thank you!
